pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -1182,18 +1230,59 @@ func deleteAllBlockerPodFinalizer(r *ReconcileComputeNodeOpenStack, instance *co
 		if err != nil && !errors.IsNotFound(err) {
 			return err
 		}
-                log.Info(fmt.Sprintf("Deleting draining jobs for node: %s", node.Name))
-                err = deleteJobWithName(r.client, r.kclient, instance, node.Name+"-drain-job-post")
-                if err != nil && !errors.IsNotFound(err) {
-                        return err
-                }
+		log.Info(fmt.Sprintf("Deleting draining jobs for node: %s", node.Name))
+		err = deleteJobWithName(r.client, r.kclient, instance, node.Name+"-drain-job-post")
+		if err != nil && !errors.IsNotFound(err) {
+			return err
+		}
 
 		// delete blocker pods to proceed with node removal from OCP
-                log.Info(fmt.Sprintf("Deleting blocker pod finalizer for node: %s", node.Name))
-                err = deleteBlockerPodFinalizer(r.client, node.Name, instance)
-                if err != nil {
-                        return err
-                }
-        }
-        return nil
+		log.Info(fmt.Sprintf("Deleting blocker pod finalizer for node: %s", node.Name))
+		err = deleteBlockerPodFinalizer(r.client, node.Name, instance)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func deleteOwnerRefLabeledObjects(r *ReconcileComputeNodeOpenStack, instance *computenodev1alpha1.ComputeNodeOpenStack) error {
```

Comment:
worth to have a comment at the beginning of this method detailing what resources are expected to be deleted (daemonsets, secrets, ...) as this list may grow soon

Reply:
done